packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -123,6 +123,28 @@ void main() {
     expect(_called, true);
   });
 
+  testWidgets('onChanged callbacks are called', (WidgetTester tester) async {
+    bool _called = false;
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: Center(
+            child: TextFormField(
+              onChanged: (String value) {
+                _called = true;
+              },
+            ),
+          ),
+        ),
+      ),
+    );
+
+    await tester.enterText(find.byType(TextField), 'Soup');
+    await tester.pump();
+    expect(_called, true);
```

Comment:
Could you also verify that the `onChanged` callback is called with the correct value (in this case, `'Soup'`)?

Reply:
Made it test for that, and it passes.